module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -452,127 +451,128 @@ namespace module::behaviour::strategy {
 
-    bool SoccerStrategy::ballDistance(const Ball& ball) {
+    bool SoccerStrategy::ball_distance(const Ball& ball) {
         return ball.position.norm() != 0.0;
     }
 
-    void SoccerStrategy::find(const std::vector<FieldTarget>& fieldObjects) {
+    void SoccerStrategy::find(const std::vector<FieldTarget>& field_objects) {
         // Create the soccer object priority pointer and initialise each value to 0.
-        auto soccerObjectPriority  = std::make_unique<SoccerObjectPriority>();
-        soccerObjectPriority->ball = 0;
-        soccerObjectPriority->goal = 0;
-        soccerObjectPriority->line = 0;
-        for (const auto& fieldObject : fieldObjects) {
-            switch (fieldObject.target.value) {
+        auto soccer_object_priority  = std::make_unique<SoccerObjectPriority>();
+        soccer_object_priority->ball = 0;
+        soccer_object_priority->goal = 0;
+        soccer_object_priority->line = 0;
+        for (const auto& field_object : field_objects) {
+            switch (field_object.target.value) {
                 case FieldTarget::Target::SELF: {
-                    soccerObjectPriority->goal        = 1;
-                    soccerObjectPriority->search_type = SearchType::GOAL_SEARCH;
+                    soccer_object_priority->goal        = 1;
+                    soccer_object_priority->search_type = SearchType::GOAL_SEARCH;
 
                     break;
                 }
                 case FieldTarget::Target::BALL: {
-                    soccerObjectPriority->ball        = 1;
-                    soccerObjectPriority->search_type = SearchType::LOST;
+                    soccer_object_priority->ball        = 1;
+                    soccer_object_priority->search_type = SearchType::LOST;
                     break;
                 }
                 default: throw std::runtime_error("Soccer strategy attempted to find a bad object");
             }
         }
-        emit(std::move(soccerObjectPriority));
+        emit(std::move(soccer_object_priority));
     }
 
-    Eigen::Vector2d SoccerStrategy::getKickPlan(const Field& field, const FieldDescription& fieldDescription) {
+    Eigen::Vector2d SoccerStrategy::get_kick_plan(const Field& field, const FieldDescription& field_description) {
         // Defines the box within in which the kick target is changed from the centre
         // of the oppposition goal to the perpendicular distance from the robot to the goal
 
-        const float maxKickRange =
+        const float max_kick_range =
             0.6;  // TODO(BehaviourTeam): make configurable, only want to change at the last kick to avoid smart goalies
-        const float xTakeOverBox = maxKickRange;
-        const float error        = 0.05;
-        const float buffer       = error + 2.0f * fieldDescription.ball_radius;          // 15cm
-        const float yTakeOverBox = fieldDescription.dimensions.goal_width / 2 - buffer;  // 90-15 = 75cm
+        const float x_take_over_box = max_kick_range;
```

Comment:
This is just essentially acting as an alias for max_kick_range at the moment, is this going to be changed to another value in future?